### PR TITLE
Add instructions for getting started with development

### PR DESCRIPTION
**Pull request by martyngigg**
The previous instructions were a little vague at how to get setup. Add a guide with more steps in it.

### CODE REVIEW

File: DevelopmentSetup.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+We recommend using [Docker Desktop](https://www.docker.com/products/docker-desktop/).
+*Please note that Docker Desktop can only be used for free under the conditions listed
+by their license agreement: <https://www.docker.com/products/docker-desktop/>.*.
+
+Download the correct package from the
+[Desktop section](https://docs.docker.com/engine/install/#desktop) of the Docker installation reference and install it.
+
+**Windows Non-Admin Setup**: For Windows an extra step is necessary to be able to run Docker without administrator privileges:
+
+* In the start menu, type `mmc` to get the management console. On the right-side click run as administrator, this should prompt you for your administrator credentials.
```
Comment (robertapplin) [inline DevelopmentSetup.md:29]: `mmc.` worked for me

**Review by sf1919 — COMMENTED** [review]
The Development Setup instructions look really good and they incorporate much of the issues we tripped up on during our session earlier. The only comment/suggested change I have is about making clear that you can minimise (but not close) Docker app when it's running and use Git Bash or another bash terminal instead. We do mention using Git Bash on a number of occasions (and thank you for making that clear) but not the fact we just want Docker running in the background.

File: README.md [hunk 2]
```diff
@@ -1,98 +1,173 @@
...
+```
+bin/shutdown.sh
+```
+
+which ensures that the webdata volume is cleaned up.
+
+Working with Django
+-------------------
+
+The first time you create a database you will need to create a Django admin account to access the Django admin interface. To do so first get into the docker container by running `docker exec -it <docker-web-container-name> bash` and then run `python manage.py createsuperuser`.
```
Comment (DanielMurphy22) [inline README.md:73]:
```suggestion
`docker exec -it <docker-web-container-name> bash` and then run `python manage.py createsuperuser`.
```

Can we keep some of this superuser stuff in either the README or DevSetup?

**Comment by SilkeSchomann** [comment]
Could we also add something about problems with windows/linux-style file endings and how to fix them? It is likely to occur more often.

**Comment by martyngigg** [comment]
> Could we also add something about problems with windows/linux-style file endings and how to fix them? It is likely to occur more often.

This should be fixed in #54.

File: README.md (hunk 2 above)
Comment (martyngigg) [reply README.md:73]: Yes. Will do. I commented this old stuff out as I didn't have everything ready in time for Monday.

I wonder if we can do this automatically when running django for the first time.

**Comment by martyngigg** [comment]
I finally got around to addressing these comments! 

Daniel is on leave. Would someone else be happy to approve?

**Review by robertapplin — APPROVED** [review]
Looks good to me 👍

**Pull request merged**